src/coprocessor_v2/endpoint.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
 
 use coprocessor_plugin_api::*;
-use kvproto::kvrpcpb;
+use kvproto::kvrpcpb as raw_coppb;
```

Comment:
And what is the purpose of this alias?

Reply:
these were introduced in previous prs. not mine

Reply:
sorry, it was me. fixing